src/applications/common/schemaform/save-in-progress/RoutedSavableReviewPage.jsx

```diff
@@ -92,6 +95,18 @@ class RoutedSavableReviewPage extends React.Component {
     );
   }
 
+  renderDowntime = (downtime = {}, children) => {
+    if (downtime.status === serviceStatus.down) {
+      const Message = this.props.formConfig.downtime.message || DowntimeMessage;
+
+      return (
+        <Message downtime={downtime}/>
+      );
+    }
+
+    return children;
+  }
```

Comment:
We ended up always providing always providing a [downtime](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/platform/monitoring/DowntimeNotification/containers/DowntimeNotification.jsx#L60) object as that first argument, it's just empty if there isn't any data.